docs/videos/healthcare-and-life-sciences/hcls-blueprint.yaml

```diff
@@ -0,0 +1,330 @@
+    source: ./community/modules/file-system/cloud-storage-bucket
+    settings:
+      name_prefix: $(vars.bucket_name_software)
+
+  - id: bucket-input-create
+    source: ./community/modules/file-system/cloud-storage-bucket
+    settings:
+      name_prefix: $(vars.bucket_name_inputs)
+
+  - id: bucket-output-create
+    source: ./community/modules/file-system/cloud-storage-bucket
+    settings:
+      name_prefix: $(vars.bucket_name_outputs)
+
+- group: software_installation
+  modules:
+
+  - id: network-install  # This will reuse network created in setup group
+    source: modules/network/pre-existing-vpc
+
+  - id: appsfs-install
+    source: modules/file-system/pre-existing-network-storage
+    settings:
+      server_ip: $(vars.appsfs_server_ip)
+      remote_mount: apps
+      local_mount: /apps
+      fs_type: nfs
+
+  - id: bucket-input-install
+    source: modules/file-system/pre-existing-network-storage
+    settings:
+      remote_mount: $(vars.bucket_name_inputs)
+      local_mount: /data_input
+      fs_type: gcsfuse
+      mount_options: defaults,_netdev,implicit_dirs,allow_other,dir_mode=0777,file_mode=766
+
+  ### Software ###
+
+  - id: spack
+    source: community/modules/scripts/spack-install
+    settings:
+      install_dir: /apps/spack
+      configs:
+      - type: file
+        scope: defaults
+        content: |
+          config:
+            build_stage:
+              - /apps/spack/spack-stage
+      - type: file
+        scope: defaults
+        content: |
+          modules:
+            default:
+              tcl:
+                hash_length: 0
+                all:
+                  conflict:
+                    - '{name}'
+                projections:
+                  all: '{name}/{version}-{compiler.name}-{compiler.version}'
+      - type: file
+        scope: site
+        content: |
+          packages:
+            slurm:
+              externals:
+                - spec: slurm@21-08-8-2
+                  prefix: /usr/local
+              buildable: False
+      compilers: [gcc@9.3.0 target=x86_64]
+      environments:
+      - name: gromacs
+        type: file
+        content: |
+          spack:
+            definitions:
+            - compilers:
+              - gcc@9.3.0
+            - mpis:
+              - intel-mpi@2018.4.274
+            - mpi_packages:
+              - gromacs@2020.5
+            specs:
+            - matrix:
+              - - $mpis
+              - - $%compilers
+            - matrix:
+              - - $mpi_packages
+              - - $%compilers
+              - - $^mpis
+
+  - id: spack-builder-startup
+    source: modules/scripts/startup-script
+    settings:
+      runners:
+      - $(spack.install_spack_deps_runner)
+      - $(spack.install_spack_runner)
+      - type: shell
+        destination: data_staging.sh
+        content: |
+          #!/bin/bash
+          wget --no-verbose -P /data_input/protien_data_bank/ https://files.rcsb.org/download/1FJS.pdb
```

Comment:
s/protien_data_bank/protein_data_bank/ (typo)

Reply:
also I do not see this pdb being used anywhere. Is it?

Reply:
This definitely deserves more explanation. The protein is the basis for a much longer example that will be demoed. The sbatch for this workflow will be submitted in a subsequent PR. The decision as of now is to not include this other sbatch as part of the blueprint and instead have it live alongside the blueprint in a separate file that is referenced by a tutorial.

We discussed removing the water example but the thought was that it is useful to have a simple example included in the blueprint. I intend do document all of this as we add meat to the tutorial. With this PR I am really just trying to get a basis from which those involved can collaborate and iterate. LMK if you would prefer to have more complete documentation in this PR.

PS. Will also correct the typo.

Reply:
The typo is fixed.